src/lib/Components/Consignments/Screens/Overview.tsx

```diff
@@ -118,18 +120,39 @@ export default class Info extends React.Component<Props, State> {
       updateSubmission(this.state, this.state.submission_id)
     } else if (this.state.artist) {
       const submission = await createSubmission(this.state)
-      this.setState({ submission_id: submission.id })
+      this.setState({ submission_id: submission.id }, this.submissionDraftCreated)
     }
   }
 
+  @track((props, state) => ({
+    action_type: Schema.ActionTypes.Success,
+    action_name: Schema.ActionNames.ConsignmentDraftCreated,
+    owner_id: state.submission_id,
+    owner_type: Schema.OwnerEntityTypes.Consignment,
+    owner_slug: state.submission_id,
+  }))
+  // tslint:disable-next-line:no-empty
+  submissionDraftCreated() {}
+
   submitFinalSubmission = async (setup: ConsignmentSetup) => {
     this.setState(setup, async () => {
       await this.updateLocalStateAndMetaphysics()
       await AsyncStorage.removeItem(consignmentsStateKey)
+      this.submissionDraftSubmitted()
       this.exitModal()
     })
   }
 
+  @track((props, state) => ({
+    action_type: Schema.ActionTypes.Success,
+    action_name: Schema.ActionNames.ConsignmentSubmitted,
+    owner_id: state.submission_id,
+    owner_type: Schema.OwnerEntityTypes.Consignment,
+    owner_slug: state.submission_id,
+  }))
+  // tslint:disable-next-line:no-empty
+  submissionDraftSubmitted() {}
```

Comment:
does this get sent? I'd assume also not?

Reply:
Nope